jwst/grism_lib/polyclip.py

```diff
@@ -0,0 +1,45 @@
+import os
+from glob import glob
+import numpy.ctypeslib as npct
+import numpy as np
+import ctypes
+from ctypes import c_int
+
+this_path = os.path.split(__file__)[0]
+#so_file = os.path.join(this_path, 'polyclip_c.cpython-39-darwin.so')
+so_file = glob(os.path.join(this_path, 'polyclip_c*.so'))[0]
+polyclip = ctypes.cdll.LoadLibrary(so_file)
```

Comment:
One should just be able to do

```python
from . import polyclip_c
```

without having to do anything with `ctypes.cdll.LoadLibrary()` here.  I think.